metrics/web/lib/dashboard/presentation/pages/dashboard_page.dart

```diff
@@ -31,6 +31,7 @@ class _DashboardPageState extends State<DashboardPage> {
     _projectMetricsNotifier =
         Provider.of<ProjectMetricsNotifier>(context, listen: false);
 
+    _projectMetricsNotifier.resetProjectNameFilter();
```

Comment:
I think we should just move it to the initState() as it seem to be unrelated to the subscription to the project errors.